<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change</class>
 <widget class="QWidget" name="Change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Редактировать</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>711</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите термин, который нужно отредактировать.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Термин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="termEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>411</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>500</y>
     <width>350</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить изменнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lpages">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Страницы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pagesEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>411</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>761</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Заполните поле!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="success">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>409</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Термин успешно изменен!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
